plugins/linux-apt/main.cpp

```diff
@@ -194,7 +193,11 @@ DownloadResult SimpleDownload(const std::string& url, const std::string& destFil
         tempPath = GenerateUniqueCachePath();
         LogDebug("Using temp file %s for %s", tempPath.data(), destFilePath.data());
 
-        msdo::download::download_url_to_path(url, tempPath);
+        std::error_code errorCode = msdo::download::download_url_to_path(url, tempPath);
+        if (errorCode)
+        {
+            throw DOPluginException("Download hit an error: %d, %s", errorCode.value(), errorCode.message().c_str());
```

Comment:
nit: string messages add bloat to the binary. Keep terse but understandable, e.g.

`"Download hit an error: %d, %s"` -> `"download_url_to_path error: %d, %s"`

Reply:
similarly on 215.